index.html

```diff
@@ -3830,7 +3832,8 @@ <h3>The <dfn>start()</dfn> method</h3>
             Whenever a new <a>Thing Description</a> |td:ThingDescription| is discovered by the underlying platform, run the following sub-steps:
             <ol>
               <li>
-                <a href="https://fetch.spec.whatwg.org/#fetch-api">Fetch</a> |td| as a JSON object |json|. If that fails, set the <a href="#dom-thingdiscovery-error">error</a> property to {{SyntaxError}}, discard |td| and continue the discovery process.
+                Retrieve |td| as a JSON object using the <a>Protocol Binding</a> specified by the |filter|'s <dfn>url</dfn> property.
```

Comment:
I would add a note that previously this was using Fetch, which remains a possibility, depending on binding.

Reply:
I added the line in 704191d2fa0925d3bbe632c57131cfd05583c934 for now, let me know if you see potential for improvement :)

Reply:
Looks good!